### DISCUSSION TITLE
Bale is blocking the tractor with baler

### DISCUSSION THREAD

**Discussion by WeAreV3N0M** [Courseplay/Courseplay_FS22#2226] [Q&A] [answered]
Any idea how could I convince CP to NOT stop when a bale is in front of the tractor? So annoying that helper wouldn't finish the job because bales blocking the path. Thats why I have a bale pusher attached at front.

**Comment by BalticHawk** [comment]
There is no way you can prevent CP from stopping. It‘s supposed to work like this.

**Reply by WeAreV3N0M** [reply]
I have managed it before so there's gonna be a way. I just can't remember. It was some weird setup.

**Reply by Tensuko** [reply]
As Tritura said, that is intended.
The only place where you could have a bale in front of you is in the turn.
My suggestion is always to start the swath on the lanes and not on the headland so when you bale, you won't have bales on the headland during turn.
Ofc when it is about baling straw, that won't work unless you do an extra swath job.

**Reply by WeAreV3N0M** [reply]
It's a contract job to bale and wrap grass (same machine, one process). After resetting the tractor the settings all went back to default and now it stops for every bale. DIDN'T STOP before reset so there is a way to achieve it.

**Reply by Tensuko** [reply]
> DIDN'T STOP before reset so there is a way to achieve it.

There is no option to turn off the object sensing.

**Reply by WeAreV3N0M** [reply]
Could you please add an exclusion to this? For example if CP identifies the blocking object as any kind of BALE, then it just continues to drive forward. It would be a useful addition.
If you make it an optional feature, people could turn it on and off at will. Balling jobs could be done with less problem, and tractors being stuck on top of bales could be avoided with a bale pusher attached to the front.
Cotton harvesters would have no such problem, as they can just push bales out of way.

**Reply by Tensuko** [reply]
Have to think about this....

**Reply by WeAreV3N0M** [reply]
> Have to think about this....

![bálázó hiba 1](https://user-images.githubusercontent.com/54554640/210611448-95d481e8-cde3-4cd3-808f-d43bcdeb2474.png)
![bálázó hiba 2](https://user-images.githubusercontent.com/54554640/210611480-04709b21-45f9-4f6d-b20d-007bebc9f2cb.png)

I think this is a problem now. On the picture there is a bale already finished but still inside the baler.
CP wouldn't start the reversing maneuver because it thinks the bale inside the machine is a blocking object.
After removing the bale with an autoload trailer, the tractor with baler started the reverse maneuver.
Just for the record: Yes, that baler is a mod and I've been using it for a while now but NEVER had this problem before.
There must have been a change after CP version 7.1.2.0 because I had no problem with that version.

**Reply by Tensuko** [reply]
that is already known and also ways arround it (no sharp corners and skip rows).
As I said, I need to think about a solution that fits.

And that "problem" does exist for a long time:
#1863

**Comment by Tensuko** [comment, marked as answer]
There is now an option to use a bale pusher and also add your own to the vehicleconfig.xml
